### ISSUE TITLE
ImportError: cannot import name 'xla_data_pb2' - Jax, Broken on Colab after current release

### ISSUE THREAD

**Issue by Tenoke** [google/jax#2436]
I use jax via trax, and today I got an error `AttributeError: 'jaxlib.tpu_client_extension.TpuExecutable' object has no attribute 'ExecutePerReplica'`

normally jax in Colab I install it using:
```
!pip install --upgrade -q jax

import requests
import os
if 'TPU_DRIVER_MODE' not in globals():
  url = 'http://' + os.environ['COLAB_TPU_ADDR'].split(':')[0] + ':8475/requestversion/tpu_driver0.1-dev20191206'
  resp = requests.post(url)
  TPU_DRIVER_MODE = 1

# The following is required to use TPU Driver as JAX's backend.
from jax.config import config
config.FLAGS.jax_xla_backend = "tpu_driver"
config.FLAGS.jax_backend_target = "grpc://" + os.environ['COLAB_TPU_ADDR']
```

However, when I saw there has been a new release I rolled back to the old one. After doing that via `!pip install --upgrade -q jax==0.1.39  jaxlib===0.1.39` I started getting `ImportError: cannot import name 'xla_data_pb2'` on `import config from jax` which I fixed temporarily by doing `url = 'http://' + os.environ['COLAB_TPU_ADDR'].split(':')[0] + ':8475/requestversion/tpu_driver_nightly'`. However, then I started getting an error on `import jax` instead

I tried doing `!pip upgrade pip` but the best I could reach is `ImportError: cannot import name 'xla_client'` on `import jax`

And again, I can install the current jax but that doesn't work with my version of trax, and I cannot install the older version of jax anymore at all.

**Comment by gnecula** [comment]
We have just released jaxlib 0.1.41 yesterday, perhaps the Colab backend is not picking up that version? 

Can you try:
```
!pip install --upgrade -q jax jaxlib
from jax import version as jax_version
print(jax_version.__version__)
from jaxlib import version as jaxlib_version
print(jaxlib_version.__version__)
```

I get 0.1.59 and 0.1.41.

Note that `jax` and `jaxlib` use separate version numbers, it is not correct to use `!pip install --upgrade -q jax==0.1.39 jaxlib===0.1.39`

**Comment by Tenoke** [comment]
I get the same, however, trax doesn't work properly with the current version. I specifically went with 0.1.39 because they are both versions for both. What versions can you recommend? Note - it was working on the weekend without pinning, just `--upgrade jax jaxlib`.

**Comment by hawkinsp** [comment]
Hmm... There might still be an issue here. I think we might need to make a new `jax` release.

Can you try with the jaxlib 0.1.41 and
* jax 0.1.59 (current) --- this does not work?
* and then from github head --- this should work?
```
pip uninstall jax
pip install --upgrade -q git+https://github.com/google/jax
```

Thanks!

**Comment by Tenoke** [comment]
```
!pip uninstall jax
!pip install --upgrade -q git+https://github.com/google/jax
!pip install --upgrade jaxlib
```

The current master index works with trax properly, while using the most recent version of jaxlib.

**Comment by hawkinsp** [comment]
Thanks! That answers the question: we just need to make a new `jax` release. I'll do that.

**Comment by hawkinsp** [comment]
Done. I released jax v0.1.60 to pypi, which should fix the problem. Hope that helps!

**Comment by Tenoke** [comment]
I can confirm, it works as expected with the most recent versions of jax and jaxlib from pip.
